app.rb

```diff
@@ -0,0 +1,141 @@
+require_relative 'student'
+require_relative 'classroom'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'print_utils'
+
+class App
+  attr_accessor :input, :person_type, :students, :persons, :books, :rentals
+
+  def initialize
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍